NPM package lacks dist

Hi there, thanks for your hard work :-).

Could you please generate the `dist` files and upload a package to NPM? The package doesn't work out of the box, it has to be built.

> Yep, I get `Cannot find module 'html-webpack-change-assets-extension-plugin'`

Could you please give me more information about this error? Like how you use it or where you want to use it?

--- a/package.json
+++ b/package.json
@@ -37,6 +37,7 @@
     "format": "yarn prettier --write",
     "format:ci": "yarn prettier --list-different",
     "lint": "tslint  --project tsconfig.json -t codeFrame 'src/**/*.ts' 'test/**/*.ts'",
+    "prepublishOnly": "npm run test && npm run build",
     "report-coverage": "cat ./coverage/lcov.info | coveralls",
     "semantic-release": "semantic-release",
     "semantic-release-prepare": "ts-node tools/semantic-release-prepare",

